scripts/categorizer.py

```diff
@@ -387,15 +395,13 @@ def main():
         """
         # Skip over if comment lines... we might need to rethink this, but it avoids problems in categorizing the rest..
         if comment_lines[line_number]:
-            categorizations.append(LineAndCats(line=line, multiline_statement_number=0, comment=1,
-                                               conditional=0, empty=0, func_def=0, assignment=0, func_call=0))
+            categorizations.append(LineAndCats(line=line, comment=1))
```

Comment:
If `line` is the first parameter, there is no need to specify it in the call. Providing the parameters can be helpful to understand what a given variable is associated it to in the parameters of the call, but in some other cases it may seem redundant (e.g. when the variable name and the parameter have the same spelling).

Reply:
;) good catch, thanks!